README.md

```diff
@@ -106,7 +106,7 @@ help.
 
 Direct integration of the Ladybug has been done in the Ladybug test webapp:
 
-- https://github.com/ibissource/ibis-ladybug-test-webapp
+- https://github.com/ibissource/ladybug-test-webapp
```

Comment:
Should be wearefrank

Reply:
Ik zag inderdaad dat ik ze nog niet allemaal had gehad. Ik heb nu overal waar mogelijk het woord "ibissource" vervangen.